android/src/main/res/values/strings.xml

```diff
@@ -12,4 +12,8 @@
 
     <string name="start_survey">Start Survey</string>
     <string name="submit_button">Submit</string>
+    <string name="exit_warning_title">Warning!</string>
+    <string name="exit_detail_message">Are you sure you want to quit the survey</string>
```

Comment:
@suho Missing the `?` :harold:

![image](https://user-images.githubusercontent.com/6950766/208823191-f806c6fc-45ad-4886-95ee-adc39e4684f2.png)

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/122/commits/c1ecbbaf030b5f34a2041a03badf77dec7a09315